### DISCUSSION TITLE
Newbie - HACS WYZE Integration won't install

### DISCUSSION THREAD

**Discussion by wreston** [SecKatie/ha-wyzeapi#304] [Q&A] [answered]
I'm a newbie running hass.io, trying to install the WYZE integration via HACS. HACS appears to be installed correctly (I'm able to add other custom integrations), but when I try to add the WYZE integration, it goes through all the expected prompts, but never actually downloads / installs. When using ssh to check the contents of custom_components folder, there is no folder for the WYZE integration. 

Can someone advise how I can get the WYZE integration installed? Thanks

**Comment by SecKatie** [comment, marked as answer]
Hey, @wreston I have recorded a video showing the process: https://cln.sh/mxSgoZ

Let me know if you need any more help!

**Reply by 3Necromancer** [reply]
I have a similar problem. 
Added HACS, 
Added repository to HACS integration, restarted HA, but wyze is not found in the HA integration list. 
Tried to uninstall the repository and reinstall (+HA restart), same result. 

Version core-2021.12.8 
Wyze version 0.1.4

**Reply by Merril-r** [reply]
I followed your video but I still do not see the integration. 

The integration downloads and I can see the files in the custom_components folder. But after a reboot I do not see the integration so that I can set it up.

Any suggestions why this is?

**Reply by SecKatie** [reply]
You have to go to the integrations page and add it just like any other UI configurable integration.

**Reply by marchare3** [reply]
the instructions say to put the wyze email in the Username field, but what is it?

**Reply by marchare3** [reply]
I figured it out.

**Comment by wreston** [comment]
Many thanks for your response. I was never able to get to as far as this screen when selecting the Wyze integration.




I ended up creating a new hass.io image, installing HACS and then the Wyze integration. The integration installed as expected on the new image.

Thanks for your support,
Walt Reston



On Dec 11, 2021, at 1:01 PM, Joshua Mulliken ***@***.***> wrote:


Hey, @wreston <https://github.com/wreston> I have recorded a video showing the process: https://cln.sh/mxSgoZ <https://cln.sh/mxSgoZ>
Let me know if you need any more help!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/JoshuaMulliken/ha-wyzeapi/discussions/304#discussioncomment-1791209>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AOYEA3K56FFJLERDBXG5ZPTUQOGX3ANCNFSM5J2H3UOQ>.
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Merril-r** [comment]
It does not show up in the list of integrations when i want to add it
